### PR TITLE
cloud: support scale up for dedicated tiers

### PR DESCRIPTION
**Pull request by ran-huang**
### What is changed, added or deleted? (Required)

add doc for [DM-5710](https://jira.tidbcloud.com/browse/DM-5710)

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [ ] master (the latest development version)
- [ ] v6.4 (TiDB 6.4 versions)
- [ ] v6.3 (TiDB 6.3 versions)
- [x] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### CODE REVIEW

File: tidb-cloud/scale-tidb-cluster.md [hunk 1]
```diff
@@ -109,16 +109,26 @@ For a running cluster, AWS and Google Cloud do not allow in-place storage capaci
...
+
+- Method 1: Increase the node size in the TiDB Cloud console
+
+    1. In the TiDB Cloud console, navigate to the **Clusters** page for your project.
+    2. Locate the cluster that you want to scale up, and click **...** in the upper-right corner of the cluster area.
+    3. Click **Modify** in the drop-down menu. The **Modify Cluster** page is displayed.
+    4. On the **Modify Cluster** page, increase the node size as you need.
+    5. Click **Confirm**.
+
+- Method 2: Increase the node size through backup and restore
```
Comment (ran-huang) [inline tidb-cloud/scale-tidb-cluster.md:124]: Are Method 2 (backup and restore) and Method 3 (support tickets) still necessary? 

IMO we can safely remove these methods now that users can scale up right in the console.

Comment (qiancai) [reply tidb-cloud/scale-tidb-cluster.md:124]: +1

File: tidb-cloud/scale-tidb-cluster.md [hunk 2]
```diff
@@ -109,9 +109,12 @@ For a running cluster, AWS and Google Cloud do not allow in-place storage capaci
 
 ## Increase node size
```
Comment (SunRunAway) [inline tidb-cloud/scale-tidb-cluster.md:110]: Release note:
Support [increasing the Node Size](https://docs.pingcap.com/tidbcloud/scale-tidb-cluster#increase-node-size)(vCPU + RAM), aka scale-up, for TiDB, TiKV, and TiFlash node of clusters created after November 30, 2022.

File: tidb-cloud/scale-tidb-cluster.md [hunk 3]
```diff
@@ -109,9 +109,12 @@ For a running cluster, AWS and Google Cloud do not allow in-place storage capaci
...
```
Comment (SunRunAway) [inline tidb-cloud/scale-tidb-cluster.md:111]: 增加一个注意吧：
```
AWS clusters only, and clusters created after 2022/yy/mm can be increased the node size.
```

Comment (SunRunAway) [reply tidb-cloud/scale-tidb-cluster.md:111]: @ran-huang

File: tidb-cloud/scale-tidb-cluster.md [hunk 4]
```diff
@@ -109,9 +109,16 @@ For a running cluster, AWS and Google Cloud do not allow in-place storage capaci
...
 
-Currently, when a cluster is running, you cannot increase its node size. You can only increase the node size through backup and restore.
+> **Note:**
+>
+> Increasing ndoe size is only available to clusters that are created in AWS after 2022/yy/mm.
```
Comment (SunRunAway) [inline tidb-cloud/scale-tidb-cluster.md:114]: Increase node storage 里有一个说明：

> AWS has a cooldown period of node storage changes. If your TiDB cluster is hosted by AWS, after changing the node storage of TiKV or TiFlash, you must wait at least six hours before you can change it again.

这个cooldown是被 Increase node storage 和 Increase node size 共享的。改了任意一个都要停6小时。

Comment (ran-huang) [reply tidb-cloud/scale-tidb-cluster.md:114]: @SunRunAway updated the notes.

File: tidb-cloud/scale-tidb-cluster.md [hunk 5]
```diff
@@ -101,17 +101,24 @@ You can also increase the storage of a TiKV or TiFlash node using TiDB Cloud API
 
 > **Note:**
 >
-> AWS has a cooldown period of node storage changes. If your TiDB cluster is hosted by AWS, after changing the node storage of TiKV or TiFlash, you must wait at least six hours before you can change it again.
+> AWS has a cooldown period of node storage or size changes. If your TiDB cluster is hosted by AWS, after changing the node storage or node size of TiKV or TiFlash, you must wait at least six hours before you can change it again.
```
Comment (SunRunAway) [inline tidb-cloud/scale-tidb-cluster.md:104]: This node should also be mentioned in "Increase node size"

Comment (SunRunAway) [inline tidb-cloud/scale-tidb-cluster.md:104]:
```suggestion
> AWS has a cooldown period of node storage changes. If your TiDB cluster is hosted on AWS, after changing the node storage or node size of TiKV or TiFlash, you must wait at least six hours before you can change it again.
```

Comment (ran-huang) [reply tidb-cloud/scale-tidb-cluster.md:104]: added in https://github.com/pingcap/docs/pull/10821/files#r1055154173

File: tidb-cloud/scale-tidb-cluster.md [hunk 6]
```diff
@@ -101,17 +101,25 @@ You can also increase the storage of a TiKV or TiFlash node using TiDB Cloud API
...
 
 For a running cluster, AWS and Google Cloud do not allow in-place storage capacity downgrade.
 
 ## Increase node size
 
-Currently, when a cluster is running, you cannot increase its node size. You can only increase the node size through backup and restore.
+> **Note:**
+>
+> - Increasing node size is only available to clusters that are created in AWS after 2022/yy/mm.
+> - AWS has a cooldown period of node storage or size changes. If your TiDB cluster is hosted by AWS, after changing the node storage or node size of TiKV or TiFlash, you must wait at least six hours before you can change it again.
```
Comment (ran-huang) [inline tidb-cloud/scale-tidb-cluster.md:115]:
```suggestion
> - AWS has a cooldown period of node size changes. If your TiDB cluster is hosted on AWS, after changing the node storage or node size of TiKV or TiFlash, you must wait at least six hours before you can change it again.
```

File: tidb-cloud/scale-tidb-cluster.md [hunk 7]
```diff
@@ -101,17 +101,25 @@ You can also increase the storage of a TiKV or TiFlash node using TiDB Cloud API
...
 ### Decrease node storage
 
 For a running cluster, AWS and Google Cloud do not allow in-place storage capacity downgrade.
 
 ## Increase node size
 
-Currently, when a cluster is running, you cannot increase its node size. You can only increase the node size through backup and restore.
+> **Note:**
+>
+> - Increasing node size is only available to clusters that are created in AWS after 2022/yy/mm.
```
Comment (qiancai) [inline tidb-cloud/scale-tidb-cluster.md:114]:
```suggestion
> - Increasing node size is only available to clusters that are hosted on AWS and created after 2022/yy/mm.
```

File: tidb-cloud/scale-tidb-cluster.md [hunk 8]
```diff
@@ -101,17 +101,25 @@ You can also increase the storage of a TiKV or TiFlash node using TiDB Cloud API
...
 ### Decrease node storage
 
 For a running cluster, AWS and Google Cloud do not allow in-place storage capacity downgrade.
 
 ## Increase node size
 
-Currently, when a cluster is running, you cannot increase its node size. You can only increase the node size through backup and restore.
+> **Note:**
+>
+> - Increasing node size is only available to clusters that are hosted on AWS and created after 2022/yy/mm.
```
Comment (ran-huang) [inline tidb-cloud/scale-tidb-cluster.md:114]:
```suggestion
> - Increasing node size is only available to clusters that are hosted on AWS and created after 2022/12/31.
```

File: tidb-cloud/scale-tidb-cluster.md [hunk 9]
```diff
@@ -101,17 +101,25 @@ You can also increase the storage of a TiKV or TiFlash node using TiDB Cloud API
...
+
+You can increase the node size for TiDB, TiKV, and TiFlash. Decreasing the node size is not supported.
 
-You need to [create a latest backup of the cluster](/tidb-cloud/backup-and-restore.md#manual-backup), [delete the cluster](/tidb-cloud/delete-tidb-cluster.md), and then increase the node size when you [restore the deleted cluster](/tidb-cloud/backup-and-restore.md#restore-a-deleted-cluster). Before taking this method, make sure the following impacts are acceptable:
+To increase the node size, take the following steps:
 
-- To avoid any data loss during or after the backup, you need to stop the connection to the cluster through your SQL client before creating the backup.
-- After you stop the connection to the cluster, your applications running on this cluster cannot provide service normally until the restoring process is completed.
+1. In the TiDB Cloud console, navigate to the **Clusters** page for your project.
+2. Locate the cluster that you want to scale up, and click **...** in the upper-right corner of the cluster area.
```
Comment (qiancai) [inline tidb-cloud/scale-tidb-cluster.md:122]:
```suggestion
2. In the row of the cluster that you want to scale, click **...**.
```

**Review by qiancai — APPROVED** [review]
other LGTM

**Review by SunRunAway — APPROVED** [review]

**Pull request merged**